telegram/ext/updater.py

```diff
@@ -38,19 +38,29 @@
     overload,
 )
 
-from telegram import Bot, TelegramError
-from telegram.error import InvalidToken, RetryAfter, TimedOut, Unauthorized
+from telegram import Bot
+from telegram.error import InvalidToken, RetryAfter, TimedOut, Unauthorized, TelegramError
 from telegram.ext import Dispatcher, JobQueue, ContextTypes, ExtBot
-from telegram.utils.warnings import PTBDeprecationWarning, warn
-from telegram.utils.helpers import get_signal_name, DEFAULT_FALSE, DefaultValue
-from telegram.utils.request import Request
+from telegram.warnings import PTBDeprecationWarning
+from telegram.request import Request
+from telegram.utils.defaultvalue import DEFAULT_FALSE, DefaultValue
+from telegram.utils.warnings import warn
 from telegram.ext.utils.types import CCT, UD, CD, BD
 from telegram.ext.utils.webhookhandler import WebhookAppClass, WebhookServer
 
 if TYPE_CHECKING:
     from telegram.ext import BasePersistence, Defaults, CallbackContext
 
 
+# From https://stackoverflow.com/questions/2549939/get-signal-names-from-numbers-in-python
```

Comment:
I was unsure, because that's not documented and because of the comment about pylint. But it seems that it's indeed just not documented, but it does exist:

* https://github.com/python/typeshed/pull/555#issuecomment-247874222
* https://bugs.python.org/issue28206

There was a [PR](https://github.com/python/cpython/pull/1939) which never made it for some reason - maybe I'll pick it up 😄 

TL;DR: will change